exams/tasks.py

```diff
@@ -129,6 +139,15 @@ def export_exam_authorizations(self):
         tsv.flush()
 
         try:
+            audit.ExamDataAuditor().audit_request(tsv.name)
+        except ImproperlyConfigured:
+            log.exception('Exam auditing improperly configured')
+            return
+        except:  # pylint: disable=bare-except
+            log.exception('Unexpected error auditing EAD file')
+            return
+
+        try:
```

Comment:
lots of repetition in this file, but it looks like that existed before this PR and could/should be addressed separately